keps/sig-node/3288-separate-stdout-from-stderr/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: Split Stdout and Stderr Log Stream of Container
+kep-number: 3288
+authors:
+  - "@knight42"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+  - sig-apimachinery
+  - sig-cli
+status: provisional
+creation-date: 2022-05-01
+reviewers:
+  - "@sig-node-leads"
```

Comment:
please add myself and @mrunalp as reviewers

Reply:
I have updated the reviewers.